3-Authorization-II/2-call-api-b2c/README.md

```diff
@@ -1,193 +1,226 @@
+1. Find the key `policyName` and replace it with your policy name e.g. `B2C_1_susi_v2`
 
-### Register the spa app (msal-react-spa)
+#### Register the client app (ms-identity-react-c3s2-spa)
 
-1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD B2C** service.
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure Active Directory B2C** service.
 1. Select the **App Registrations** blade on the left, then select **New registration**.
 1. In the **Register an application page** that appears, enter your application's registration information:
-   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-react-spa`.
-   - Under **Supported account types**, select **Accounts in any identity provider or organizational directory (for authenticating users with user flows)**.
-   - In the **Redirect URI (optional)** section, select **Single-page application** in the combo-box and enter the following redirect URI: `http://localhost:3000/`.
-1. Select **Register** to create the application.
-1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
-1. Select **Save** to save your changes.
-1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
-   - Select the **Add a permission** button and then:
-       - Ensure that the **My APIs** tab is selected.
-       - In the list of APIs, select the API `msal-react-api`.
-       - In the **Delegated permissions** section, select the **Access 'msal-react-api'** in the list. Use the search box if necessary.
-       - Select the **Add permissions** button at the bottom.
-   - Finally, select the **Grant admin consent** button at the top.
-
-#### Configure the spa app (msal-react-spa) to use your app registration
+    1. In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `ms-identity-react-c3s2-spa`.
+    1. Under **Supported account types**, select **Accounts in any identity provider or organizational directory (for authenticating users with user flows)**
+    1. Select **Register** to create the application.
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Single-page application** option.
+    1. In the **Redirect URI** section enter the following redirect URIs:
+        1. `http://localhost:3000`
+        1. `http://localhost:3000/redirect.html`
+    1. Click **Save** to save your changes.
```

Comment:
check why code gen didnt generate this part?

Reply:
updated in sample.json